packages/salesforcedx-vscode-core/package.json

```diff
@@ -807,6 +807,11 @@
           "type": "boolean",
           "default": true,
           "description": "%experimental_apex_library_description%"
+        },
+        "salesforcedx-vscode-core.experimental.supportFunctions": {
```

Comment:
The only "problem" I see with defining the flag like this right now is that users will be able to see it and enable it. We do want them to do that once we have all the changes ready but for now we might want to "hide" this by just defining the flag as part of `sfdxCoreSettings.ts`. This is what we do with the `internal` flag.

https://github.com/forcedotcom/salesforcedx-vscode/blob/6e97314cbc8492de0defe220dadfa4417d4b5b3e/packages/salesforcedx-vscode-core/src/settings/sfdxCoreSettings.ts#L66